app/experimenter/docs/adrs/0002-use-react-bootstrap-cra-gql-for-nimbus.md

```diff
@@ -0,0 +1,113 @@
+
+### Negative Consequences
+
+This approach for `nimbus-ui` will require time to reach parity to the current `rapid` project. 
+
+## Pros and Cons of the Considered Alternatives
+
+### Refactor existing project
+
+* Pros
+  * Less time spent on new project scaffolding
+* Cons
+  * Because the `rapid` project is small, this may cause points of frustration and more time than expected shuffling files around, refactoring, modifying tests, and reconnecting the pieces with a new global state management approach compared to starting anew
+  * We desire to use the build pipeline offered and managed by `create-react-app` regardless (see [FxA's React Toolchain ADR](https://github.com/mozilla/fxa/blob/main/docs/adr/0013-react-toolchain-for-settings-redesign.md)) and it may be awkward porting an existing application over
+  * Refactoring instead of "starting fresh" provides a weaker sense of collective ownership from new contributors and excitement for fresh perspectives
+
+### Don't use GQL
+
+* Pros
+  * Less additional work on the front-end and back-end initially
+  * Non-FxA engineers would likely experience a learning curve switching to GQL
+* Cons
+  * As the application grows, we will likely need to set up Redux or grow the current ad-hoc global state management
+  * We won't reap the benefits of a GQL application (see (FxA's GQL ADR)[https://github.com/mozilla/fxa/blob/main/docs/adr/0016-use-graphql-and-apollo-for-settings-redesign.md]) such as reduced network requests and data transferred, clarity around expected data and server responses, client-side caching and global state management via `apollo-client-cache`, helpful developer tooling, and speedier development in the long run
+
+### Use Tailwind instead of Bootstrap
+
+* Bootstrap
+  * Pros
+    * Offers a complete UI kit/library to aid in accelerated component styling, leading to less time spent on a custom UI and less time needed on style polishing
+    * Is already being used in the `rapid` project, leading to less component refactoring. With a new `nimbus-ui` application, bits and pieces can likely be copied over
+  * Cons
+    * Isn’t being used in FxA and prevents any potential React component sharing down the line with FxA if we make `fxa-react` more widely available
+    * Is another CSS framework and style pattern for newcomers to pick up on and has a steeper learning curve than other CSS solutions because it offers more
+    * Is a little bulky - we won’t use everything it offers, but this is a much less important consideration for an internal tool like Nimbus
+    * Lacks customization options without forking our own copy or adding style overrides
+  * Tailwind
```

Comment:
Is this unintentionally nested under Bootstrap?

Reply:
Nope, just a spacing mishap :P